plugins/techdocs/src/reader/transformers/html/hooks/iframes.ts

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
maybe rename file `removeUnsafeIframes`, could be clearer?

Reply:
Let me give you some context, I used _iframe_ in the filename because this file contains everything related to iframes, including the `removeUnsafeIframes` function, and others: `isSafe` and `isIframe`.
Does it make sense to you?

Reply:
Yes, makes sense!